ddl/backfilling.go

```diff
@@ -320,15 +325,14 @@ func (w *backfillWorker) updateLease(execID string, bfJob *BackfillJob, nextKey
 }
 
 func (w *backfillWorker) finishJob(bfJob *BackfillJob) error {
-	bfJob.State = model.JobStateDone
 	return w.backfiller.FinishTask(bfJob)
 }
 
 func (w *backfillWorker) String() string {
-	if w.backfiller == nil {
-		return fmt.Sprintf("worker %d", w.id)
+	if w.backfiller == nil || w.GetCtx() == nil {
+		return ""
```

Comment:
What kind of scenario will this happen? Here the return is “” could help to identify the problem?